awslambda/putsnippet/index.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+console.log('Loading function');
+
+const aws = require('aws-sdk');
+
+const s3 = new aws.S3({ apiVersion: '2006-03-01' });
+
+
+exports.handler = (event, context, callback) => {
+    console.log(JSON.stringify(event));
+    let user_id = event.pathParameters.user_id;
```

Comment:
You used camelCase for this variable in all the other functions, use it here for consistency.